airdrop/application/services/user_registration_services.py

```diff
@@ -134,13 +134,10 @@ def register(self, inputs):
             is_registered_user = self.is_elgible_registered_user(
                 airdrop_window_id, address)
 
-            if is_registered_user is not None:
-                raise Exception(
-                    "Address is already registered for this airdrop window")
-
-            UserRepository().register_user(airdrop_window_id, address)
+            if is_registered_user is None:
+                UserRepository().register_user(airdrop_window_id, address)
```

Comment:
What happens if the user is already registered for a given AirDrop Window ?
and tries to re register again ?

Reply:
It returns a success response only.  If we throw a 4xx series status code, in the UI we cannot redirect to the Airdrop window claims.

Reply:
ok , lets check how the envision is handling this and we need to be in synch with it